CHANGES.rst

```diff
@@ -151,6 +151,11 @@ astropy.utils
 
 - ``JsonCustomEncoder`` is expanded to handle ``Quantity`` and ``UnitBase``.
   [#5471]
+- ``astropy.utils``
+
+  - Added ``dcip_xy`` method to IERS
+	The data for the CIP offsets is now available for use
+	in coordinate frame conversion
```

Comment:
Also this needs ``[#5837]`` at the end to indicate the Pull Request it came from.